apiutils/task.go

```diff
@@ -0,0 +1,21 @@
+package apiutils
```

Comment:
`api/equality` would be okay but we need to define this particular notion of equality. Removing meta and status is a very specific kind of equality.

"A little copying is better than a little dependency."